samples/ehr-service/lambda/auth/index.js

```diff
@@ -41,25 +58,40 @@ function authorizeRequest(userToken, serviceToken, path, method, methodArn, call
     console.log('auth service response time - auth public endpoint: ' + latency + 'ms')
     let decoded = ''
     if (resp.status == 200) {
-      decoded = jwt.decode(userToken)
-      callback(null, generatePolicy(decoded.sub, 'Allow', methodArn, decoded.accountId, latency))
+      if (!userToken) {
+        // The path accessed without token found in exclusion list
+        // hence ARC returned 200.
+        callback(null, generatePolicy("USER", "Allow", methodArn, "", latency));
+      } else {
+        // A valid access token was found
+        let decoded = jwt.decode(userToken)
```

Comment:
This eventually needs to go in our Node/Python SDK. I don't want developers to go to two/three different places to deal with tokens. The fragmented state of identity is what we are unifying and simplifying.